src/main/java/org/aion/wallet/ui/MainWindow.java

```diff
@@ -40,8 +40,8 @@
     private static final String TITLE = "Aion Wallet";
     private static final String MAIN_WINDOW_FXML = "MainWindow.fxml";
     private static final String AION_LOGO = "components/icons/aion-icon.png";
-
-    private final BlockchainConnector blockchainConnector = BlockchainConnector.getInstance();
+    private static final String LOCAL_DIR = System.getProperty("user.dir");
```

Comment:
Couldn't this be local.storage.dir ?

Reply:
actually no, beacuse this is where the app is started from, so we need this